test/common/perf_monitor_test.cpp

```diff
@@ -17,6 +17,7 @@ class PerfMonitorTests : public TerrierTest {
     common::PerfMonitor monitor(false);
 
     auto db_main = terrier::DBMain::Builder().SetUseGC(true).SetUseCatalog(true).Build();
+    monitor.Start();
```

Comment:
@mbutrovich I had to add this to get my test to pass. Is this fine?

Reply:
Yep!